schedule/yast/maintenance/offline_migration_15sp4.yaml

```diff
@@ -0,0 +1,28 @@
+name: schedule/yast/maintenance/migration_offline.xml
```

Comment:
the name of a test is not an .xml file path.

Reply:
changed.